docs/outlook/debug-autolaunch.md

```diff
@@ -12,7 +12,7 @@ This article discusses the key debugging stages to enable and set breakpoints in
 
 To begin debugging, select the tab for your applicable client.
 
-# [Windows](#tab/windows)
+# [Windows (classic)](#tab/windows)
```

Comment:
Let's also ensure that tabs are ordered: web (modern then classic) -> Windows (new then classic) -> Mac -> mobile (Android then iOS) as applicable. E.g., for this page, that would be: Web -> Windows (new) -> Windows (classic) -> Mac. Here and other instances